apps/telehealth-widget/README.md

```diff
@@ -0,0 +1,323 @@
+
+|User scenario|Service flow|
+|--|--|
+|A doctor and a patient sign up and sign in to the widget. |The solution uses [Amazon Cognito](https://aws.amazon.com/cognito/) for authentication and authorization. When the user signs up, the solution creates an Amazon Chime SDK `AppInstanceUser`.
+|A doctor creates an appointment with a patient. |The solution creates an [Amazon Chime SDK messaging](https://docs.aws.amazon.com/chime-sdk/latest/dg/using-the-messaging-sdk.html) channel and adds two users as `ChannelModerator`. 
+|A doctor and a patient view current and upcoming appointments. |The solution lists Amazon Chime SDK messaging channels moderated by the user.
+|A doctor and a patient check in for an appointment. |The solution uses [Amazon Chime SDK messaging channel flows](https://docs.aws.amazon.com/chime/latest/dg/using-channel-flows.html) to set the user's presence status to "Checked in."
+|A doctor and a patient join a video call. |The solution initiates an [Amazon Chime SDK meeting](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html) with two Amazon Chime SDK attendees.
+|A patient receives an SMS notification if a patient has not checked in. |The solution uses [Amazon Pinpoint](https://aws.amazon.com/pinpoint/) to send an SMS notification.|
+|A doctor and a patient join a meeting with a PSTN call. |The solution uses [Amazon Chime SDK PSTN Audio service](https://docs.aws.amazon.com/chime-sdk/latest/dg/build-lambdas-for-sip-sdk.html) to make a PSTN outbound call from doctor to patient.|
+
+## Demo screenshots
+
+### Sign in/Sign up
+
+![authentication](./github-assets/auth.png)
+
+### Appointment
+
+![appointment](./github-assets/appointment.png)
+
+### Check-in and SMS notification
+
+![checkin-sms-notifcation](./github-assets/sms-notification.png)
+
+### Audio video call (Doctor)
+
+![audio-video-meeting-doctor](./github-assets/audio-video-meeting.png)
+
+### Audio video call (Patient)
+
+![audio-video-meeting-patient](./github-assets/audip-video-meeting-patient.png)
+
+### Phone call (Doctor)
+
+![phone-call-doctor](./github-assets/phone-call-doctor.png)
+
+### Phone call (Patient)
+
+![phone-call-patient](./github-assets/phone-call-patient.png)
+
+### Chat widget
+
+![chat](./github-assets/chat.png)
+
+
+## Solution Architecture
+
+![solution architecture](./github-assets/widget-architecture.png)
+
+## Getting started
+
+Ensure that you have configured your workstation with the default AWS credentials and an AWS region. Your default credentials should have the administrator permissions to deploy the stack and upload files to an Amazon S3 bucket. See [Getting started with the AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites) if you have not configured so.
+
+### Sending appointment reminders via SMS
```

Comment:
I understand that this section is required to send an SMS messages, but the section has lots of steps. How about putting this section between **Running the demo** and **Clean up** because the last step **7-iii** already requires builders to deploy it again.

As a result, customers simply do `npm run deploy` to get started with.

Reply:
Yes agreed, moved as suggested.